server/service/src/log_service.rs

```diff
@@ -31,9 +36,17 @@ pub trait LogServiceTrait: Send + Sync {
             Some(file_name) => file_name,
             None => default_filename,
         };
-
         let log_file_path = log_dir_path.join(&file_name);
-        let log_file_content = fs::read_to_string(log_file_path)?;
+
+        let log_file_content = if file_name.ends_with(".gz") {
```

Comment:
Hmm I wonder if we need to test this on Android ? 

I did a quick check and simple-log uses log4rs, which uses flate2 or compression, I see you also use flate2, it looks like it should be fine for Android (it's just rust code)